english/styles/styles_ctrl.pcss

```
@@ -1,3 +1,4 @@
+@import '../../shared/components/ui/use_of_funds/variables.pcss';
```

there is an import missing here to make it look good:
```suggestion
@import '../../shared/components/ui/use_of_funds/variables.pcss';
....

@import '../../shared/components/ui/use_of_funds/FundsContent.pcss';
```